lms/djangoapps/branding/api.py

```diff
@@ -0,0 +1,214 @@
+                "title": unicode(settings.SOCIAL_MEDIA_FOOTER_DISPLAY.get(social_name, {}).get("title", "")),
+                "url": settings.SOCIAL_MEDIA_FOOTER_URLS.get(social_name, "#"),
+                "social_icon": settings.SOCIAL_MEDIA_FOOTER_DISPLAY.get(social_name).get("icon", "")
+            }
+        )
+    return links
+
+
+def about_edx_link():
+    """ Returns the list of marketing links of footer
+    """
+
+    return [
+        {
+            "title": _("About"),
+            "url": marketing_link('ABOUT')
+        },
+        {
+            "title": _("News"),
+            "url": marketing_link('NEWS')
+        },
+        {
+            "title": _("Contact"),
+            "url": marketing_link('CONTACT')
+        },
+        {
+            "title": _("FAQ"),
+            "url": marketing_link('FAQ')
+        },
+        {
+            "title": _("edX Blog"),
+            "url": marketing_link('BLOG')
+        },
+        {
+            "title": _("Donate to edX"),
+            "url": marketing_link('DONATE')
+        },
+        {
+            "title": _("Jobs at edX"),
+            "url": marketing_link('JOBS')
+        }
+    ]
+
+
+def get_footer_logo(site_name, is_edx_domain):
+    """ Return the logo used for footer about link
+
+    Args:
+        site_name(str): The site url to get the absolute link
+        is_edx_domain(bool): Flag to check is it in edx domain or open edx
+
+    Returns:
+        Absolute url to logo
+    """
+    if is_edx_domain:
+        logo_file = 'images/edx-theme/edx-header-logo.png'
+    else:
+        logo_file = 'images/default-theme/logo.png'
+    try:
+        url = site_name + staticfiles_storage.url(logo_file)
+    except:
+        url = site_name + logo_file
+    return url
+
+
+def get_mobile_urls():
+    """ Returns the mobile app urls
+
+    """
+    mobile_urls = []
+    if settings.FEATURES.get('ENABLE_FOOTER_MOBILE_APP_LINKS'):
+        mobile_urls = [
+            {
+                "title": "Apple",
+                "url": settings.MOBILE_STORE_URLS.get('apple', '#'),
+                "logo_image_url": 'images/app/app_store_badge_135x40.svg'
+            },
+            {
+                "title": "Google",
+                "url": settings.MOBILE_STORE_URLS.get('google', '#'),
+                "logo_image_url": 'images/app/google_play_badge_45.png'
+            }
+        ]
+    return mobile_urls
+
+
+def get_footer_static(file_name):
+    """ Returns the static js/css contents as a string
+
+    Args:
+        file_name(str): path to the static file name under static folder
+
+    Raises:
+        I/O Error if file not found
+    Returns:
+        Contents of static file
+
+    """
+    key = "footer/{file_name}".format(file_name=file_name)
+    contents = cache.get(key)
+    if contents:
+        return contents
+    file_dir = os.path.dirname(os.path.abspath(__file__))
+    file_path = os.path.join(file_dir, "static/{}".format(file_name))
```

Comment:
You could make this more readable by importing the methods you use here from `os.path` at the top of this module.
```python
from os.path import dirname, abspath, join
```